youtube_dl/extractor/deezer.py

```diff
@@ -46,25 +45,17 @@ def _real_extract(self, url):
             webpage, 'data JSON')
         data = json.loads(data_json)
 
-        playlist_title = data.get('DATA', {}).get('TITLE')
-        playlist_uploader = data.get('DATA', {}).get('PARENT_USERNAME')
+        playlist_title = data['DATA']['TITLE']
+        playlist_uploader = data['DATA']['PARENT_USERNAME']
```

Comment:
Read coding conventions and don't touch this.

Reply:
In the for loop, the initial author used square brackets. That's why I made the confusion. I changed it as well in the last commit.